<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="2,1">
    <item>
     <layout class="QGridLayout" name="m_grdlChannels"/>
    </item>
    <item>
     <layout class="QGridLayout" name="m_grdlFourier"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="m_mainMenu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>690</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="m_mnuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="m_mnuFileOpen"/>
    <addaction name="m_mnuFileSave"/>
    <addaction name="separator"/>
    <addaction name="m_mnuFileExit"/>
   </widget>
   <addaction name="m_mnuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="m_mnuFileOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="m_mnuFileSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="m_mnuFileExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
